portfolio/src/main/webapp/index.html

```diff
@@ -68,6 +68,9 @@ <h2 id="comments">Leave your comments</h2>
         <p>Visitors' comments</p>
         <div id="comment-container"></div>
         <br/>
+        <p>Input the Maximum number of comments shown:</p>
```

Comment:
Check out the `label` tag, which labels an input semantically. It should reference the `id` of the target `input` element, and this is how you get the behavior of clicking the label in HTML and having your cursor placed in the input field. That behavior is less useful for number inputs, but it's still good to use the `label` tag.

Reply:
I added the label tag in the latest commit.